packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -262,6 +287,13 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// Where this widget should be anchored during a drag.
   final DragAnchor dragAnchor;
 
+  /// An optional injectable strategy that overrides the
```

Comment:
suggestion:
```
A strategy that is used by this draggable to get the the anchor offset when it widget is being dragged.

The anchor offset refers to the distance between the users' finger and the [feedback] widget when this draggable is being dragged.

Defaults to [defaultDragAnchorStrategy].
```

Reply:
Your text is more straigh to the point. I will change it.